### DISCUSSION TITLE
Request Collaborator Access

### DISCUSSION THREAD

**Discussion by fredchristianson** [fredchristianson/javascript-game-engine#2] [General]
Reply here if you would like to be added as a collaborator.
Include 
 - your github username.
 - name you will use in branches.  See Branch Discussion.  I am "free", so  task-0-env-**fred** is a branch I'd use.
 - any expertise you have.  We could use someone who knows GitHub better than me.  Anything else you think you can help with?

**Comment by chylinski82** [comment]
- github username: chylinski82
- name in branches: chris
- i have a basic knowlegde of github and git, use it for my own projects, never used it in an actual team. Technologies i'm most comfortable with: JS, ReactJS, CSS, HTML, chatGPT, PhaserJS also some: Python3, Redux, Firebase Firestore, Command Line, NodeJS

**Reply by fredchristianson** [reply]
Welcome Chris.  I'm new to managing github for other people but if I did it right, you are now a collaborator.  Any and everything may change the rest of today other than your collaborator status.   I'm still playing with things and working on a plan.

**Comment by chylinski82** [comment]
All set. Thanks Fred.

**Comment by Eddie7145** [comment]
GitHub Username : **Eddie7145**

Name in branches : **Chizzy**

Well I have good knowledge of git and how to use GitHub, I'm also at Css, Sass, Html and figma for now, learning JavaScript currently. With this I believe I would better myself and hopefully have a smooth and fun experience building this with you guys. 
PS: I have a very wild imagination 😉

**Comment by RimuhRimu** [comment]
`your github username.` : @RimuhRimu
`name you will use in branches. `: Rimuh
`any expertise you have. `: I've worked at various personal projects with Javascript, Typescript, (Deno/node), css, Sass, html, figma, lua, python, I've made some simple games with javascript and lua before, I've worked before with a small team building simple static websites(html,css,js)

**Comment by tommyc96** [comment]
Count me in!

**Reply by fredchristianson** [reply]
you're invited to collaborate

**Comment by fauzaanu** [comment]
Hi, I would like to participate in this in anyway I can.
github username : fauzaanu
branch : fauzaanu

I have never worked with a game / game engine before. I have a basic understanding of Git and Github. I havent used git in a team. I mainly code in JS & Python, and I have pickedup other languages fairly easily for one time projects (php, c#, c++, GO, Kotlin etc). Also a design guy so I'll just keep following until I see somewhere I can contribute to, considering I dont really understand how this would work. But please count me in. 😄

**Reply by fredchristianson** [reply]
I invited you to be a contributor.  
I'm still learning how github works from the maintainer side.  Let me know if it looks like I'm not doing it right.
Also let me know if the entire concept of tasks makes sense.  You don't have to do any, but I'm trying to break out small work that everyone can do - as much as a tutorial where everyone does the same thing as a project.  In the short term we are building some basic functionality that isn't just game related.  Also have discussions about different aspects of games so we can design the right architecture.  
I think I will try to speed things up and get to games sooner.  But still create tasks and discuss architecture for those who want it.

**Comment by cebisi** [comment]
Greetings!  I would like to participate in this in anyway I can.
github username : cebisi
name to use in a branch : Kenechi

Expertise I have: I do not have experience working with a game / game engine before. Also, I have a basic understanding of Git and Github. I have not used git in a team. I mainly code in JS & Python, and I have minimal experience with other languages. I am passionate and plan to build real projects to improve. Please be patient with me as I am more than willing to help contribute but I am not as well versed. Please count me in

**Reply by fredchristianson** [reply]
welcome.  I invited you.  I intend for this project to be about learning and gaining experience.  
Ask questions.  
Try things.
Have fun with it.
